src/inverseRobot.js

```diff
@@ -25,6 +25,26 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  const keys = Object.keys(robot);
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
instead of 2 loops see if you can do it without loops 
may be with set
check this resource
[Link](https://javascript.plainenglish.io/storing-unique-values-with-set-object-in-javascript-ac2707dbdb6)

Reply:
implemented. thanks.